### ISSUE TITLE
plugin shows All images when choosing,

### ISSUE THREAD

**Issue by aihilali** [wymsee/cordova-imagePicker#208]
every time I want to choose a picture or mutiple pictures using imagePicker plugin, it shows all the images avialable on the device at once. is there a way to make it show them by groups. I mean, first when you want to choose a picture or multiple pictures it shows the folders where the pictures are stored, (downloads, gallerry, whatsapp ...) then after choosing one of those sources the plugin shows the avialable pictures in there

**Comment by nbaua** [comment]
Yes, Even I would like to see that feature, I've been facing issue of delay when user have hundreds of photos on their mobile and imagePicker plugin shows them all. Some users might not like the UX in this case.
